test/lib/ex_aws/ec2/integration_test.exs

```diff
@@ -5,4 +5,21 @@ defmodule ExAws.EC2.IntegrationTest do
     assert {:ok, %{body: body}} = ExAws.EC2.describe_instances |> ExAws.request
     assert body |> String.contains?("DescribeInstancesResponse")
   end
+
+  test "#describe_tags" do
```

Comment:
Is it necessary to do actual requests here? Can't you just assert on the request body?